composer.json

```diff
@@ -24,6 +23,7 @@
         "ext-xhprof": "You need to install either xhprof or uprofiler to use XHGui.",
         "ext-uprofiler": "You need to install either xhprof or uprofiler to use XHGui.",
         "ext-mongo": "Mongo is needed to store profiler results for PHP < 7.",
-        "ext-mongodb": "Mongo is needed to store profiler results for PHP > 7."
+        "ext-mongodb": "Mongo is needed to store profiler results for PHP > 7.",
+        "alcaeus/mongo-php-adapter": "Mongo PHP Adapter is required for PHP >7 (when using ext-mongodb)"
```

Comment:
I'm thinking about the best way to approach this. Since composer does not allow installing packages that aren't mentioned in `composer.json`, I believe the cleanest approach would be adding `alcaeus/mongo-php-adapter` to `require-dev`, and modifying the `composer update` command in the installation script to install dev dependencies as well. What do you think?